src/openrct2/EditorObjectSelectionSession.cpp

```diff
@@ -375,6 +376,14 @@ void reset_selected_object_count_and_size()
     }
 }
 
+void finish_object_selection(uint8_t editorStep)
```

Comment:
```suggestion
void finish_object_selection(RCT2_EDITOR_STEP editorStep)
```

Reply:
It's nice that you've applied this change directly from GitHub, but you'll need to adjust the headers as well, or this won't compile.